core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -131,6 +132,43 @@ message CacheRequest {
   optional bool async = 6;
 }
 
+// Request for load a block into alluxio
+// next available id: 4
+message LoadRequest {
+  repeated File files = 1;
+  optional string user = 2;
+  optional int64 bandwidth = 3;
+}
+
+message File{
+  optional string ufs_path = 1;
+  repeated int64 block_id = 2;
+  required int64 block_size = 3;
+}
+
+message LoadResponse {
+  repeated FileStatus file_status = 1;
+  required TaskStatus status = 2;
+}
+
+message FileStatus {
+  repeated BlockStatus block_status = 1;
+}
+
+enum TaskStatus {
+  SUCCESS = 0;
+  FAILURE = 1;
+  PARTIAL_FAILURE = 2;
+}
+
+message BlockStatus {
+  // The status code, which should be an enum value of [google.rpc.Code][google.rpc.Code].
```

Comment:
Do we need to add a block id here to indicate status for that block?

Reply:
I think it would go with each block so no need to put id here causing duplication

Reply:
i see, thanks